httpd/handler.go

```diff
@@ -667,6 +673,99 @@ func (h *Handler) serveProcessContinuousQueries(w http.ResponseWriter, r *http.R
 	w.WriteHeader(http.StatusAccepted)
 }
 
+func (h *Handler) serveRunMapper(w http.ResponseWriter, r *http.Request) {
+	// we always return a 200, even if there's an error because we always include an error object
+	// that can be passed on
+	w.Header().Add("content-type", "application/json")
+	w.WriteHeader(200)
```

Comment:
If we remove it, let's fix the bug in the http logger then, otherwise the log may report status `0`.

We need to change the function here:

https://github.com/influxdb/influxdb/blob/distributed-queries/httpd/response_logger.go#L45

to be something like:

```go
func (l *responseLogger) Status() int {
	if l.status == 0 {
		l.status == http.StatusOk
	}
	return l.status
}
```